README.md

```diff
@@ -1,123 +1,81 @@
-
-``` conda create -n <virtual_env_name> python=3.7```
-
-To activate the virtual environment do
-
-``` source activate <virtual_env_ńame>```
-
-To install packages use either pip or conda: 
-
-``` conda/pip install <package_name> ```
-
-## Usage
-
-You can either call `yaml2sbml` via its [command line interface](#Command_line_interface) or within your [python code](#Python): 
-
-### Command line interface
-
-#### yaml2sbml command
-
-To convert an ODE model encoded in a yaml file to SBML using the terminal, go to the `yaml2sbml` folder and run:
+`yaml2sbml`can be `pip`-installed via
 
 ```shell
- yaml2sbml <yaml_input_file> <sbml_output_file>
+pip install yaml2sbml
 ```
+For more infos see the [docs](https://yaml2sbml.readthedocs.io/en/latest/#).
 
-For instance, using the yaml file in the examples folder:
-
-```shell
- yaml2sbml ../examples/ode_input1.yaml ../examples/sbml_out.xml
-```
+## Getting Started
 
-#### yaml2petab command
+* The documentation of the tool is hosted on [read the docs](https://yaml2sbml.readthedocs.io/en/latest/)
+* The [format documentation](https://github.com/yaml2sbml-dev/yaml2sbml/blob/main/doc/format_specification.rst) describes the input YAML. 
 
-If you want to generate PEtab parameter, observable and condition tables, additionally to the SBML file using the terminal, go to the `yaml2sbml` folderand run:
+* Jupyter notebooks containging examples can be found under [`doc/examples`](https://github.com/yaml2sbml-dev/yaml2sbml/tree/main/doc/examples).  Most notably:
+    * [Lotka_Volterra.ipynb](https://github.com/yaml2sbml-dev/yaml2sbml/tree/main/doc/examples/Lotka_Volterra/Lotka_Volterra_python/Lotka_Volterra.ipynb) showing the Python package.
+    * [Lotka_Volterra_CLI.ipynb](https://github.com/yaml2sbml-dev/yaml2sbml/tree/main/doc/examples/Lotka_Volterra/Lotka_Volterra_CLI/Lotka_Volterra_CLI.ipynb) showing the command line interface.
+    * [Lotka_Volterra_Mock.ipynb](https://github.com/yaml2sbml-dev/yaml2sbml/tree/main/doc/examples/Lotka_Volterra/Lotka_Volterra_Model_Editor/Lotka_Volterra_Model_Editor.ipynb) showing the Model Editor.
 
-```shell
- yaml2petab <yaml_input_file> <petab_output_directory> <model_name>
-```
-
-For instance, again using the yaml file in the examples folder:
-```shell
- yaml2petab ../examples/ode_input1.yaml ../examples/ example_model.xml
-```
+## Basic Syntax
 
 ### Python
 
-Alternatively you can call `yaml2sbml` within your python code via
-
+A YAML model can be translated to SBML/PEtab in Python via
 ```python
 import yaml2sbml
 
-yaml2sbml.yaml2sbml(yaml_file, sbml_file)
-```
+# SBML conversion
+yaml2sbml.yaml2sbml(input_yaml_dir, output_sbml_dir)
 
-Here all inputs ar given as strings.
-
-To generate PEtab files call `yaml2petab` via
+#PEtab conversion
+yaml2sbml.yaml2petab(input_yaml_dir, 
+                     output_petab_dir,
+                     sbml_name)
```

Comment:
The argument names here differ from the definitions, e.g. from `yaml2sbml.py` and `yaml2PEtab.py`:
```python
def yaml2sbml(yaml_file: str, sbml_file: str):
def yaml2petab(yaml_file: str,
               output_dir: str,
               model_name: str,
               petab_yaml_name: str = None,
               measurement_table_name: str = None):
```

Could be changed for consistency.

Reply:
You are right, but I guess, that the names in `yaml2petab.py` are not as descriptive as possible. Are you. in favor of changing the names `yaml2petab.py` then?

Reply:
I think it's OK. But in general, as there are "files", "filenames" and "directories", it doesn't make much sense to name the input `input_yaml_dir` (as in the README), when it's not a directory, or `yaml_file` (as in the method definition), when it's not a file object like
```python
with open('my_yaml.yml', 'r') as yaml_file:
    yaml2petab(yaml_file, ...)
```
If the method was overloaded to accept file objects as well, like the above example, then `yaml_file` makes more sense as a name for type `Union[str, TextIO]`. If kept as-is, I would opt for `yaml_filename` (but really, fine as-is in the definition).

Similarly, `model_name` is less obvious than `sbml_filename` to me (I think `sbml_filename` reflects how it is used?).

I think simply changing the usage in this README to match the method definition would be an improvement.

Reply:
You are right, probably the naming here is a bit sloppy, I would therefore rather go for renaming the names in the function definition... :)